fastly.toml

```diff
@@ -3,4 +3,6 @@ name = "Static content"
 description = "Apply performance, security and usability upgrades to static bucket services such as Google Cloud Storage or AWS S3, including redirects, headers, and a 404 page."
 authors = ["<oss@fastly.com>"]
 language = "rust"
-service_id = ""
+
+[requires]
+backends = ["bucket_origin"]
```

Comment:
**Question(s)**: 
- What does `[requires]` represent?
- How is it meant to be used?

Reply:
We talked about this and I'm backing out these changes - we'll revisit later.